lib/reverse_adoc/html_converter.rb

```diff
@@ -46,11 +46,13 @@ def self.convert(input, options = {})
              end
 
       root || (return "")
-
       ReverseAdoc.config.with(options) do
-        result = ReverseAdoc::Converters.lookup(root.name).convert(root)
-        ReverseAdoc.cleaner.tidy(result)
+        result = ReverseAdoc::Converters.lookup(root.name).to_coradoc(root)
       end
     end
+    def self.convert(input, options = {})
+      result = Coradoc::Generator.gen_adoc(to_coradoc(input, options))
+      ReverseAdoc.cleaner.tidy(result)
+    end
```

Comment:
Why do we need any cleaning after putting it into a `Coradoc::Document`? The to_adoc results should be perfect.